### PR TITLE
Fix slider slider handle alignment

### LINKED ISSUE #2710: Revisions Selector slider handle misaligned
### Expected
The Revisions Slider handle site beneath the horizontal line.

### Observed
The Revisions Slider handle site atop the middle of the horizontal line.

### Reproduced
1. Log into the Simplenote web or desktop app.
2. Open the Revisions Selector via the History button in the top-right corner.

| Chrome | Safari |
| --- | --- |
| chrome-revisions-selector-handle | safari-revisions-selector-handle |

### Where did you see the bug
- System Make: Mac
- System Model: MacBook Pro
- OS: macOS
- OS version: 11.2.2
- Browser (if applicable): Chrome
- Browser version (if applicable): 88.0.4324.192
- Simplenote app version: 2.7.0-beta1

### PR DESCRIPTION
**Pull request by dcalhoun**
### Fix
Fixes #2710. Chrome didn’t apply the flex box styles in the same way as Safari or
Electron. This resulted in a misaligned drag handle in Chrome.

These changes replace flex box styles with a negative `translateY` to
vertically center the handle. The slider track increased its height by
1px to avoid a need to split a pixel to find the vertical middle.

| Before | After |
| --- | --- |
| slider-before | slider-after |

### Test
1. Open Simplenote in Chrome.
1. Open the revision selector by clicking the History button in the top-right.

_Expected Outcome:_ The slider handle is vertically aligned in the slider track.

### Release

Fix slider handle alignment in Chrome.

### CODE REVIEW

**Review by dmsnell — COMMENTED** [review]
Looks much better!

**Comment by dcalhoun** [comment]
@dmsnell I noticed your review wasn't technically an "approval" in terms of GitHub UI. Should I await an official approval from the team before merging?

**Review by dmsnell — APPROVED** [review]

**Pull request merged**
